server/backends/redis_key_val_store/redis_key_val_store.go

```diff
@@ -0,0 +1,40 @@
+package redis_key_val_store
```

Comment:
how bout redis_kvstore? (same for memory_kvstore)

Reply:
Done